app/channels/chat_channel.rb

```diff
@@ -1,24 +1,24 @@
 class ChatChannel < ApplicationCable::Channel
   def subscribed
     if params['room'] == "local_chat"
-      stream_for ChatRoom.find_by(system: current_user.reload.system)
+      stream_for ChatRoom.local.where(system: current_user.reload.system).first
```

Comment:
Why use `where(...).first` instead of `find_by` ?

Reply:
I find the behavior more explicit.  A query where you get the first one.  I tend side with the "less magic, more explicit" side of Rails development

Reply:
I would still rather use `find_by` so its always clear that there always should be only one entry. `where(...).first` more seems like 'its okay if there are more entries matching but we will only pick the first one'. If you get what I mean 😄

Reply:
Also, there are cases where `.find_by` doesn't work and you have to use `.where().first` -- so, in my mind.. why not use one method, instead of two...

Reply:
besides... `find_by()` is doing the same SQL query (minus the ordering that `.first` does)

Reply:
Okay, sure. Can you replace all occurences of find_by then? Would be awesome!

Reply:
done